packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -54,19 +57,18 @@
           />
         </HideDuringScreenshot>
       </template>
-      <template #panel3="{width}">
+      <template #panel3>
         <HideDuringScreenshotOrRunMode class="bg-white">
           <SpecRunnerHeaderOpenMode
             v-if="props.gql.currentProject"
             :gql="props.gql.currentProject"
             :event-manager="eventManager"
             :get-aut-iframe="getAutIframeModel"
-            :width="width"
           />
         </HideDuringScreenshotOrRunMode>
 
         <RemoveClassesDuringScreenshotting
-          class="h-full bg-gray-100 p-16px"
+          class="bg-gray-100 h-0 p-16px"
```

Comment:
Doesn't seem like we need the background color here at this point. Do we need to set a height with the vertical overflow disabled now?

Reply:
Setting an explicit height (even 0) prevents scrolling in a way that I don't think users are actually able to do, but that I saw in Cypress-in-Cypress tests, and was causing a failure. It seems possible that some users might experience this as well (say, with assistive tech?), so I'm inclined to leave it in. A more general fix for this probably involves touching the `body` element based on what route the user is on, and seems to go beyond the scope of this PR.

You're right the gray is redundant though, since it's now handled in the panel itself, which was otherwise leaving a weird white area at the bottom some times after the overflow changes. So I pulled out the grays.